source/features/more-dropdown.tsx

```diff
@@ -31,10 +31,32 @@ async function init(): Promise<void> {
 		createDropdown();
 	}
 
+	let compareUrl = `/${repoUrl}/compare`;
+	let commitsUrl = `/${repoUrl}/commits`;
+	let tree = '';
```

Comment:
```suggestion
	let ref = '';
```

I think `ref` is generic `git` term for this.